app/src/main/java/gov/hhs/cdc/trustedintermediary/external/apache/ApacheClient.java

```diff
@@ -0,0 +1,63 @@
+package gov.hhs.cdc.trustedintermediary.external.apache;
+/**
+ * This class implements Httpclient and is a "humble object" for the Apache Client 5 library. Using
+ * Apache Client 5 Fluent facade, we are able to perform CRUD operations such as POST and GET, in a
+ * generic way.
+ */
+import gov.hhs.cdc.trustedintermediary.wrappers.HttpClient;
+import java.io.IOException;
+import java.util.Map;
+import org.apache.hc.client5.http.fluent.Request;
+import org.apache.hc.core5.http.Header;
+import org.apache.hc.core5.http.io.entity.StringEntity;
+import org.apache.hc.core5.http.message.BasicHeader;
+import org.jetbrains.annotations.NotNull;
+
+public class ApacheClient implements HttpClient {
+
+    private static final ApacheClient INSTANCE = new ApacheClient();
+
+    private ApacheClient() {}
+
+    public static ApacheClient getInstance() {
+        return INSTANCE;
+    }
+
+    @Override
+    public String post(
+            @NotNull String url, @NotNull Map<String, String> headerMap, @NotNull String body)
+            throws IOException {
+        Header[] headers = convertMapToHeader(headerMap);
+
+        return Request.post(url)
+                .setHeaders(headers)
+                .body(new StringEntity(body))
+                .execute()
+                .returnResponse()
+                .toString();
+    }
+
+    @Override
+    public String get(
```

Comment:
`get` is no longer used.  Let's delete it!

Reply:
done.